File: odd_even.java
import java.util.Scanner;

public class odd_even {
    public static void main(String[] args){
        Scanner input=new Scanner(System.in);
        System.out.println("enter the number for odd_even evaluation:");
        int num=input.nextInt();
        if (num%2==1){
            System.out.println(num+" is odd");
        }
        else if (num%2==0){
            System.out.println(num+" is even");
        }
    }
}

File: odd_even.cpp
#include<iostream>
using namespace std;
void fun(long long int);
int main(){
    long long int num;
    cout<<"enter the number : ";
    cin>>num;
    cout<<num;
    fun(num);
    return 0;
}
void fun(long long int num){
    
    if(num%2==0)
    
        cout<<" is even!";
    else
        cout<<" is odd!";
}